syncd/scripts/syncd_init_common.sh

```diff
@@ -24,12 +24,20 @@ fi
 # Use temporary view between init and apply
 CMD_ARGS+=" -u"
 
-case "$(cat /proc/cmdline)" in
-  *fast-reboot*)
+BOOT_TYPE="$(cat /proc/cmdline | grep -o 'SONIC_BOOT_TYPE=\S*' | cut -d'=' -f2)"
```

Comment:
>SONIC_BOOT_TYPE= [](start = 42, length = 16)

We should handle both cases for backward-compatible with 201803:

fast-reboot
SONIC_BOOT_TYPE=fast-reboot
Otherwise we cannot fast-reboot from 201803 into 201811.